### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
Many thanks to contributing to qbic-pipelines/bamtofastq!

Please fill in the appropriate checklist below (delete whatever is not relevant). These are the most common things requested on pull requests (PRs).

## PR checklist
 - [ ] This comment contains a description of changes (with reason)
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If necessary, add test data to `testdata/` 
 - [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
 - [ ] Make sure your code lints (`nf-core lint .`). 
 - [ ] Documentation in `docs` is updated
 - [ ] `CHANGELOG.md` is updated
 - [ ] `README.md` is updated

**Learn more about contributing:** https://github.com/qbic-pipelines/bamtofastq/tree/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: .github/workflows/push_dockerhub.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
           echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
           docker tag qbicpipelines/bamtofastq:latest qbicpipelines/bamtofastq:dev
           docker push qbicpipelines/bamtofastq:dev
```
Comment (SusiJo) [inline .github/workflows/push_dockerhub.yml:32]: Why is the specific release number not specified here as well?

**Review by SusiJo — COMMENTED** [review]
Looks good to me @FriederikeHanssen 🚀 
I just have a minor question on the docker pushing

**Comment by SusiJo** [comment]
The Changelog and docs were already updated in a previous PR, right?

**Comment by FriederikeHanssen** [comment]
> The Changelog and docs were already updated in a previous PR, right?

Yes, see changelog here: https://github.com/qbic-pipelines/bamtofastq/blob/dev/CHANGELOG.md

**Pull request merged**